packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -72,7 +71,6 @@ export type PendingMessageResubmitData = Pick<
 export interface IRuntimeStateHandler {
 	connected(): boolean;
 	clientId(): string | undefined;
-	close(error?: ICriticalContainerError): void;
```

Comment:
>close(error?: ICriticalContainerError): void; [](http://example.com/codeflow?start=1&length=45)

Just for my knowledge - how do we ensure that removing something from an exported interface, wouldn't break any consumers? Where all do we search for consumers?

Reply:
As long as there is no tag or only `@internal` tag, then the repo building is all the confirmation you need.

Reply:
We are removing the type validation checks for things tagged `@internal` this sprint I believe.